src/Dialogs/DataField.cpp

```diff
@@ -102,18 +102,16 @@ EditDataFieldDialog(const TCHAR *caption, DataField &df,
       auto type = AndroidTextEntryDialog::Type::TEXT;
 
       AndroidTextEntryDialog dlg;
-      auto new_value = dlg.ShowModal(Java::GetEnv(), *context,
-                                     caption, value, type);
-      if (!new_value)
-        return false;
+      auto new_value =
+          dlg.ShowModal(Java::GetEnv(), *context, caption, value, type);
+      if (!new_value) return false;
```

Comment:
Stuff like this makes it hard to review the changes, as there is no functional change.